tests/testNumberField.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Tests for the number field type.
+ *
+ * @package WordPress
+ * @subpackage Custom Meta Boxes
+ */
+
+class NumberFieldTestCase extends WP_UnitTestCase {
+
+	function setUp() {
+		parent::setUp();
+
+		$args = array(
+			'post_author'  => 1,
+			'post_status'  => 'publish',
+			'post_content' => rand_str(),
```

Comment:
Ugh of course we do. Alright, I'll open an issue to clean those up since that's a bit 💩